src/Update.elm

```diff
@@ -277,7 +277,7 @@ shipSpeed ship =
 isShipLanded : Ship -> Float -> Bool
 isShipLanded ship platformPos =
     abs ship.y
-        > 0.99
+        > 0.97
```

Comment:
Yes, the ship was going almost below the platform before stopping so I had to adjust. Could not verify as I did not have the previous version running.